lib/broccoli/angular2-app.js

```diff
@@ -55,7 +56,13 @@ Angular2App.prototype.toTree = function () {
     //     outputFile: '/app.js'
     // });
 
-    return mergeTrees([assetTree, tsSrcTree, tsTree, jsTree, this.index(), vendorJsTree], { overwrite: true });
+    var lessTree = new Funnel(sourceTree, {
+        include: ['**/*.less'],
+        allowEmpty: true
+    });
+    var lessSrcTree = compileLess(lessTree);
+
+    return mergeTrees([assetTree, tsSrcTree, tsTree, jsTree, this.index(), lessSrcTree, vendorJsTree], { overwrite: true });
```

Comment:
@wesleycho here is an example of current usage of `package.json` for storing `angular-cli` config: https://github.com/angular/angular-cli/pull/134#issuecomment-174208779